components/match2/wikis/marvelrivals/get_match_group_copy_paste_wiki.lua

```diff
@@ -0,0 +1,49 @@
+---
+-- @Liquipedia
+-- wiki=marvelrivals
+-- page=Module:GetMatchGroupCopyPaste/wiki
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+
+local BaseCopyPaste = Lua.import('Module:GetMatchGroupCopyPaste/wiki/Base')
+
+---@class MarvelRivalsMatch2CopyPaste: Match2CopyPasteBase
+local WikiCopyPaste = Class.new(BaseCopyPaste)
+
+local INDENT = WikiCopyPaste.Indent
+
+--returns the Code for a Match, depending on the input
+---@param bestof integer
+---@param mode string
+---@param index integer
+---@param opponents integer
+---@param args table
+---@return string
+function WikiCopyPaste.getMatchCode(bestof, mode, index, opponents, args)
+	local showScore = Logic.nilOr(Logic.readBool(args.score), bestof == 0)
+	local opponent = WikiCopyPaste.getOpponent(mode, showScore)
+
+	local lines = Array.extendWith({},
+		'{{Match',
+		Logic.readBool(args.needsWinner) and (INDENT .. '|winner=') or nil,
```

Comment:
I rather have finished input over winner. Winner can always be calculated from the score, and finished is an easier input (harder to make typo)